api/v1/openapi.yaml

```diff
@@ -1742,5 +1758,20 @@ definitions:
       last-error:
         description: Last error seen while performing desired action
         type: string
+  Metric:
+    description: Metric map
```

Comment:
This description field is misleading; change it to "metric data" - metric map implies that there are multiple metrics in this field.

Reply:
There are multiple metrics in one map.. I don't see why it should be singular?

Reply:
@joestringer in this case, the description says 'metric map', but this is just one metric. The API returns an array of Metric - this was the source of my confusion.

Reply:
There is only one metric. /metrics has the option array.

Reply:
https://github.com/cilium/cilium/blob/bf522df233bed59ef23db0e062987c3784b13807/api/v1/openapi.yaml#L661